templates/rhpam71-kieserver-externaldb.yaml

```diff
@@ -151,22 +151,68 @@ parameters:
   name: KIE_SERVER_CONTROLLER_PORT
   example: '8080'
   required: false
-- displayName: KIE Server Persistence DS
-  description: KIE server persistence datasource (Sets the org.kie.server.persistence.ds system property)
-  name: KIE_SERVER_PERSISTENCE_DS
-  value: java:/jboss/datasources/rhpam
+- displayName: KIE server controller protocol
+  description: KIE server controller protocol (Used to set the org.kie.server.controller system property)
+  name: KIE_SERVER_CONTROLLER_PROTOCOL
+  example: http
   required: false
+- displayName: KIE Server controller token
+  description: KIE server controller token for bearer authentication (Sets the org.kie.server.controller.token system property)
+  name: KIE_SERVER_CONTROLLER_TOKEN
+  required: false
+#    {
+#      "displayName": "KIE Server Persistence Schema",
+#      "description": "Hibernate persistence schema.",
+#      "name": "KIE_SERVER_PERSISTENCE_SCHEMA",
```

Comment:
Even though these lines are commented, please make them YAML instead of JSON format.